rust_dev_preview/s3/README.md

```diff
@@ -1,20 +1,28 @@
-# AWS SDK for Rust code examples for Amazon S3
+# Amazon S3 examples using SDK for Rust
```

Comment:
Amazon S3 code examples for the SDK for Rust

Reply:
Fixed.